posts/views.py

```diff
@@ -69,6 +81,13 @@ def post_create(request):
         save_hashtags(parsedString['hashtags'], post)
     return redirect(request.META['HTTP_REFERER'] or 'landingpage')
 
+def set_post_replies(post):
```

Comment:
Because it get s called with `set_post_replies(context['post'], self.request)` in line [98](https://github.com/7Pros/circuit/pull/142/files#diff-7f5d848739da3904921a61c635a3254bR98).

This method should not exist at all, it should be integrated with `set_post_extra`.